daemon/info.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-func (daemon *Daemon) CmdInfo(job *engine.Job) error {
+func (daemon *Daemon) Info() (*engine.Env, error) {
```

Comment:
Even if the result is just an array of strings, let's either create a new type in api/types or just use `[]string` instead of engine.Env because I'm hoping engine.Env goes away, eventually.